app/models/data_resources/resource_modules/public_transportation/travel_time.rb

```diff
@@ -0,0 +1,128 @@
+ # get travel times for a specific date
+# GTFS field descriptions: https://developers.google.com/transit/gtfs?hl=de
+#
+# @param [String] date "2023-09-18T12:00"
+#
+# @return [JSON] [
+#   {
+#     "stop_sequence": "26",
+#     "stop_headsign": "",
+#     "pickup_type": "0",
+#     "drop_off_type": "0",
+#     "arrival_time": "12:06:00",
+#     "departure_time": "12:06:00",
+#     "trip": {
+#       "trip_headsign": "Name der Endhaltestelle",
+#       "trip_short_name": "00772",
+#       "direction_id": "0",
+#       "wheelchair_accessible": "1",
+#       "bikes_allowed": "0"
+#     },
+#     "route": {
+#       "route_short_name": "24",
+#       "route_long_name": "",
+#       "route_type": "3",
+#       "route_color": "951b81",
+#       "route_text_color": "ffffff",
+#       "route_desc": ""
+#     }
+#   }
+# ]
+class PublicTransportation::TravelTime
+  def initialize(date:, external_id:, data_provider_id:)
+    @external_id = external_id
+    @date = date
+    @travel_date = DateTime.parse(@date)
```

Comment:
`@travel_date` is only used in the `initialize`, so it does not require the global `@`. i will clean that up

Reply:
done with 81be479